src/KM_Controls.pas

```diff
@@ -1826,7 +1884,7 @@ constructor TKMPanel.Create(aParent: TKMMasterControl; aLeft, aTop, aWidth, aHei
   inherited Create(nil, aLeft, aTop, aWidth, aHeight);
 
   fMasterControl := aParent;
-  aParent.fCtrl := Self;
+  aParent.fMasterPanel := Self;

```

Comment:
1. Renamed TKMMasterControl.fCtrl into TKMMasterControl.fMasterPanel as it means exactly master panel (main panel)